projects/storefrontapp/src/index.html

```diff
@@ -81,6 +81,11 @@
       src="webApplicationInjector.js"
       data-smartedit-allow-origin="localhost:9002"
     ></script>
+
+    <!--BEGIN QUALTRICS WEBSITE FEEDBACK SNIPPET-->
+    <script type="text/javascript" src="qualtricsIntegration.js"></script>
```

Comment:
I'd still prefer to load this lower level, to make sure it's lazily loaded. It's currently blocking the initial page and might not be needed at the first page load. This is going to affect the experience badly.